app/models/setting/content.rb

```diff
@@ -114,8 +114,6 @@ def self.default_settings
                N_('Default Location where new Puppet content will be put upon Content View publish'),
                nil, N_('Default Location Puppet content'), nil,
                :collection => proc { Hash[Location.unscoped.all.map { |loc| [loc[:title], loc[:title]] }] }),
-      self.set('host_update_lock', N_("Allow multiple concurrent Actions::Katello::Host::Update calls for one host to be processed at the same time."),
```

Comment:
This probably also needs a migration to clean up the existing setting from the db

Reply:
https://github.com/Katello/katello/pull/8502